src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -215,13 +215,9 @@ const PersonalDetails = (() => {
         if (!is_error) {
             const redirect_url = localStorage.getItem('personal_details_redirect');
             // to update tax information message for financial clients
-            BinarySocket.send({ get_account_status: 1 }, { forced: true }).then((response_status) => {
-                showHideTaxMessage();
-                Header.displayAccountStatus();
-                if (redirect_url && +response_status.get_account_status.prompt_client_to_authenticate && Client.isAccountOfType('financial')) {
-                    $('#msg_authenticate').setVisibility(1);
-                }
-            });
+            showHideTaxMessage();
+            Header.displayAccountStatus();
```

Comment:
we need get_account_status for these two, please keep the deleted line

Reply:
Should be okay if I just remove the response_status parameter? Otherwise the linter screams at me.